samples/bluetooth/mesh/light/child_image/mcuboot/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# MCUboot config to enable multi core update with secondary slot in external flash
+CONFIG_PM=n
+
+CONFIG_MAIN_STACK_SIZE=10240
+CONFIG_MBEDTLS_CFG_FILE="mcuboot-mbedtls-cfg.h"
+
+CONFIG_BOOT_SWAP_SAVE_ENCTLV=n
+CONFIG_BOOT_ENCRYPT_IMAGE=n
+
+CONFIG_BOOT_UPGRADE_ONLY=n
+CONFIG_BOOT_BOOTSTRAP=n
+
+# Enable QSPI drivers for external flash
+CONFIG_NORDIC_QSPI_NOR=y
+CONFIG_BOOT_MAX_IMG_SECTORS=256
+
+# Logging
+CONFIG_LOG=y
+CONFIG_MCUBOOT_LOG_LEVEL_INF=y
+CONFIG_LOG_MODE_MINIMAL=y
+
+# Enable simultaenous multi-core updates
+CONFIG_NRF53_MULTI_IMAGE_UPDATE=y
+CONFIG_UPDATEABLE_IMAGE_NUMBER=2
+CONFIG_BOOT_UPGRADE_ONLY=y
+CONFIG_PCD_APP=y
+
+# Dependencies for CONFIG_NRF53_MULTI_IMAGE_UPDATE
+CONFIG_FLASH=y
+CONFIG_FPROTECT=y
+CONFIG_FLASH_SIMULATOR=y
```

Comment:
Flash simulator. What is it, and why do we need it?

Reply:
It is used by the nRF5340 platform to simulate the primary MCUBOOT slot for the network core. As i have understood it, the MCUBOOT in the application core can not access the flash owned by the network core directly, so it has to be put in RAM in order to transfer it to the network core.